### PR TITLE
Changed var name

### PR DESCRIPTION
**Pull request by TomArcherMsft**

### CODE REVIEW

**Comment by TomArcherMsft** [comment]
@grayzu Could you please review and merge this? I renamed some variables for consistency.

File: quickstart/101-analysis-services-create/variables.tf [hunk 1]
```diff
@@ -10,7 +10,7 @@ variable "resource_group_name_prefix" {
   description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
 }
 
-variable "azurerm_analysis_services_server_name_prefix" {
+variable "analysis_services_server_name_prefix" {
```
Comment (lonegunmanb) [inline quickstart/101-analysis-services-create/variables.tf]: This variable is an unused variable now, I think we should just delete it.

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
I think we've got an unused variable, would you please just delete it? @TomArcherMsft

File: quickstart/101-analysis-services-create/variables.tf (hunk 1 above)
Comment (TomArcherMsft) [reply quickstart/101-analysis-services-create/variables.tf]: Thanks. Fixed.

**Comment by TomArcherMsft** [comment]
@lonegunmanb I've made the edits. Thanks for the review!

**Comment by grayzu** [comment]
lgtm

**Pull request merged**